maisim/maisim.Game.Tests/Visual/Component/TestSceneTrackCardFocusedManyStyle.cs

```diff
@@ -1,42 +1,95 @@
-﻿using maisim.Game.Component;
+﻿using maisim.Game.Beatmaps;
+using maisim.Game.Component;
 using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
 using osu.Framework.Testing;
 using osuTK;
 
-namespace maisim.Game.Tests.Visual.Component
+namespace maisim.Game.Tests.Visual.Component;
+
+public class TestSceneTrackCardFocusedManyStyle : GridTestScene
 {
-    public class TestSceneTrackCardFocusedManyStyle : GridTestScene
+    public TestSceneTrackCardFocusedManyStyle() : base(2, 3)
     {
-        public TestSceneTrackCardFocusedManyStyle() : base(2,2)
+        Cell(0, 0).Children = new Drawable[]
+        {
+            new SpriteText
+            {
+                Anchor = Anchor.TopLeft,
+                Origin = Anchor.TopLeft,
+                Text = "DifficultyRating.Basic",
```

Comment:
nit: you may want to use string formatting along with the nameof() operator instead of manually writing the name of every variant but since this very enum is bound to not change in the future i guess it can stay like that? Do as you see fit.

Reply:
Use nameof() is better. Maimai is always update so sometime it can be change in the future.